test/torch/tensors/test_paillier.py

```diff
@@ -0,0 +1,84 @@
+import pytest
+import torch as th
+from syft.frameworks.torch.tensors.interpreters import PaillierTensor
+from syft.frameworks.torch.utils.paillier import PaillierEncryption
+import syft as sy
+
+def test_hook():
+    assert sy.TorchHook(th)
+
+def test_wrap():
+    p = PaillierEncryption(length=14)
+    x_tensor = th.Tensor([1, 2, 3])
+    x = PaillierTensor(p.public_key).on(x_tensor)
+    assert isinstance(x, th.Tensor)
+    assert isinstance(x.child, PaillierTensor)
+    assert isinstance(x.child.child, th.Tensor)
+
+def test_encryption():
+    p = PaillierEncryption(length=14)
+    x_ = th.LongTensor([1,2,3])
+    y_ = th.LongTensor([2,-2,3])
+    x = PaillierTensor(p.public_key).on(x_)
+    y = PaillierTensor(p.public_key).on(y_)
+    x.child.encryptTensor(p.getEncryptedTensors())
```

Comment:
It would be ideal if we could simply add a "paillier" function to native.py such that you can take any LongTensor (such as "x_") and call x_.paillier(key) and it would automatically return a wrapped PaillierTensor.

Reply:
That's actually a great idea, I will look into it.

Reply:
Let's wait to resolve this conversation until it's in - helps me keep track :)